Gemfile.lock

```diff
@@ -358,6 +360,8 @@ GEM
     thread_safe (0.3.5)
     tilt (1.4.1)
     timecop (0.8.0)
+    typhoeus (0.8.0)
```

Comment:
You need typhoeus for that external script, but the rails app itself probably doesn't need it.  Let's go ahead and remove that from the Gemfile and re-submit.